portals/devportal/source/src/app/components/Apis/Details/SourceDownload.jsx

```diff
@@ -85,6 +85,30 @@ function SourceDownload(props) {
             });
     };
 
+    const downloadGraphQLSchema = () => {
+        const newAPI = new API();
+        const promisedGraphQL = newAPI.getGraphQLSchemaByAPIId(api.id);
+        promisedGraphQL.then((response) => {
+            const windowUrl = window.URL || window.webkitURL;
+            const binary = new Blob([response.data]);
+            const url = windowUrl.createObjectURL(binary);
+            const anchor = document.createElement('a');
```

Comment:
@HiranyaKavishani 
We have this [Util function](https://github.com/wso2/apim-apps/blob/main/portals/publisher/source/src/app/data/Utils.js#L365-L402) to do the same, Can't we use that ?